.gitignore

```diff
@@ -0,0 +1,10 @@
+# Ignore PlatformID and Visual Code files that aren't needed
```

Comment:
PlatformID —> PlatformIO ?

Reply:
Hah, missed that. If you squint hard enough the `D` becomes an `O`. I will get that fixed.